docs/getting-started-k8s.md

```diff
@@ -29,5 +29,5 @@ helm repo add aws-sb https://awsservicebroker.s3.amazonaws.com/charts
 helm inspect aws-sb/aws-servicebroker
```

Comment:
It doesn't look like the `helm inspect aws-sb/aws-servicebroker` command accepts `--devel`.